accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/configreader/YamlConfigReader.java

```diff
@@ -286,7 +286,10 @@ private void setupAceBean(final String principal,
                 currentAceDefinition, ACE_CONFIG_PROPERTY_PRIVILEGES));
         tmpAclBean.setPermission(getMapValueAsString(
                 currentAceDefinition, ACE_CONFIG_PROPERTY_PERMISSION));
+        // kept for backward compatibility reasons
         tmpAclBean.setRepGlob((String) currentAceDefinition.get(ACE_CONFIG_PROPERTY_GLOB));
```

Comment:
we should only allow the old syntax, but in our code get as quick as possible to the new model => tmpAclBean.loadRestrictionsFrom(currentAceDefinition) should add ACE_CONFIG_PROPERTY_GLOB as "rep:glob" to restrictionMap in AceBean